File: 2_JAVA/Assignment 6/Loan.java
package Loan;

public abstract class Loan{
	

	public double emi(double principle,int period){

     

		double total =0;
		
		total = principle * (1+rate()*period/100)/(12*period);


		return total;
	}


	public abstract double rate();
	
	protected double principle;
	protected int period;


}

File: Assignment5/DemoApp/loan.cs
namespace Met.Loan;

public  abstract class Loan
{
	

	public double Emi(double principle,int period)
	{

		double total =0;
		
		total = principle * (1+Rate()*period/100)/(12*period);


		return total;
	}


	public abstract double Rate();
	
	protected double principle;
	protected int period;


}
